src/Permanent/MediumPermanentPlainTextFormatter.php

```diff
@@ -83,10 +83,18 @@ private function generateWeekScheme(array $openingHoursData): string
 
         $weekScheme = $this->getWeekScheme($weekDaysOpen, $this->formatter);
 
+        // Put all the day names with opening hours on a single line with 'Open at' (sec) at the beginning.
+        // E.g. 'Open at mo & th - su'
+        if ($this->isFirstPeriodASingleDay($weekDaysOpen)) {
+            return PlainTextSummaryBuilder::start($this->translator)
+                ->openAt(...$weekScheme)
+                ->toString();
+        }
+
         // Put all the day names with opening hours on a single line with 'Open at' (sec) at the beginning.
```

Comment:
Is this comment still correct?

Reply:
In English it is, only the Dutch should be text should be split up.

Reply:
I don't think "Open at" is correct English by the way (hence the "(sec)" in the comment which is a typo and should be "(sic)" haha)

Reply:
Googling around, it seems like it should either be "open from" or "is opened at" / "opens at".